recipes/mingw-w64/linux/conanfile.py

```diff
@@ -215,18 +224,18 @@ def build(self):
                         autotools.install()
 
             self.output.info("Building libgcc ...")
-            with tools.chdir(os.path.join(self.build_folder, "gcc")):
+            with chdir(self, os.path.join(self.build_folder, "gcc")):
                 autotools_gcc.make()
                 autotools_gcc.install()
 
         self.output.info("Building done!")
 
     def package(self):
         self.copy("COPYING", src=os.path.join(self.build_folder, "sources", "mingw-w64"), dst="licenses")
-        tools.remove_files_by_mask(self.package_folder, "*.la")
-        tools.rmdir(os.path.join(self.package_folder, "share", "man"))
-        tools.rmdir(os.path.join(self.package_folder, "share", "doc"))
-        tools.rmdir(os.path.join(self.package_folder, "lib", "pkgconfig"))
+        rm(self, "*.la", self.package_folder)
```

Comment:
```suggestion
        rm(self, "*.la", self.package_folder, recursive=True)
```
It should remove those files listed by the hook.